content/blog/2023/03/09/2023-03-09-jenkins-newsletter.adoc

```diff
@@ -150,7 +150,9 @@ Additionally, build logs are now viewed from the main panel with easier navigati
 [[security-fixes]]
 image:/images/post-images/2023/01/12/jenkins-newsletter/security.png[Security Update]
 
-Contributed by: author:wadeck[Wadeck Follonier]
+Contributed by: author:kguerroudj[Kevin Guerroudj]
 
-The security team has prepared the March 8, 2023 security release of Jenkins `2.394` (weekly), Jenkins `2.387.1` (LTS), and Jenkins `2.375.4` (previous LTS).
-Special thanks to Daniel Beck and other security team members for their work on the upcoming release of Jenkins core and our plugin security reports.
+Two security advisories have been published during the month of February:
+* One regarding link:/security/advisory/2023-02-15/[plugins], including 5 plugins that were affected and have been fixed, with one vulnerable to a sandbox bypass vulnerability.
+* One regarding link:/security/advisory/2023-02-09/[Docker images] and fixing the vulnerabilities https://github.com/git/git/security/advisories/GHSA-c738-c5qq-xg89[CVE-2022-23521] and https://github.com/git/git/security/advisories/GHSA-475x-2q3q-hvwq[CVE-2022-41903] present in git, making remote code execution possible.
```

Comment:
I guess it wasn't the correct formating to get a list or even a RF 😢 
Could you fix that @kmartens27?

![image](https://user-images.githubusercontent.com/91883215/224125437-0f54049a-c873-4d82-b0df-89fdffc55dc0.png)

Reply:
Oh my gosh, absolutely @Kevin-CB ! Sorry about not catching that previously!